### PR TITLE
Adfs Module: Improve Inputs and Outputs for the AdfsDeviceRegistrationUpnSuffix Cmdlets

### PR DESCRIPTION
**Pull request by X-Guardian**
This PR improves the descriptions for the Inputs and Outputs section of the following AdfsDeviceRegistrationUpnSuffix cmdlets:

- Add-AdfsDeviceRegistrationUpnSuffix
- Get-AdfsDeviceRegistrationUpnSuffix
- Remove-AdfsDeviceRegistrationUpnSuffix
- Set-AdfsDeviceRegistrationUpnSuffix

### CODE REVIEW

**Comment by e0i** [comment]
@X-Guardian Thank you for your contribution.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/adfs/Get-AdfsDeviceRegistrationUpnSuffix.md [hunk 1]
```diff
@@ -56,15 +56,9 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
-- SslPort: **ushort**
-- IsSetAsSslBinding: **bool**
-- IsCustom: **bool**
-- Upn: **string**
-- SslPort: **ushort**
-- IsSetAsSslBinding: **bool**
-- IsCustom: **bool**
+### Microsoft.IdentityServer.Management.Commands.GetAdfsDeviceRegistrationUpnSuffix+DrsBinding
+
+Returns one or more DrsBinding objects that represent the deevice registration service UPN suffix resources for the Federation Service.
```
Comment (get-itips) [inline docset/windows/adfs/Get-AdfsDeviceRegistrationUpnSuffix.md]:
```suggestion
Returns one or more DrsBinding objects that represent the device registration service UPN suffix resources for the Federation Service.
```

**Pull request merged**